### DISCUSSION TITLE
Cuenta marcada por github

### DISCUSSION THREAD

**Discussion by Francisco2020-Ra** [community/community#138724] [Other Feature Feedback, Questions, & Ideas]
Buenos dias Comunidad, una de mis cuentas fue "marcada" por github, luego de que sufri un hackeo en la cuenta, intento recuperarla e intente comunicarme con soporte, pero hace casi un mes aproximadamente que no me contestan. pueden darme una mano con eso?

**Comment by ghostinhershell** [comment]
Hi @Francisco2020-Ra,

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! In the future, please only open 1 ticket per incident.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
